kansha/app/comp.py

```diff
@@ -46,11 +46,11 @@ class Kansha(object):
     """The Kansha root component"""
```

Comment:
Note: too many entities in this module (`Kansha`, `MainTask`, `App` and `WSGIApp`). At least `App`, which is indeed empty, can be removed.

BTW Kansha v2 has only `Kansha` and `WSGIApp`